src/ServiceManagement/Sql/Commands.SqlDatabase/Database/Cmdlet/StartAzureSqlDatabaseExport.cs

```diff
@@ -66,7 +64,7 @@ public class StartAzureSqlDatabaseExport : SqlDatabaseCmdletBase
             ParameterSetName = ByContainerObjectParameterSet,
             HelpMessage = "The Azure Storage Container to place the blob in.")]
         [ValidateNotNull]
-        public AzureStorageContainer StorageContainer { get; set; }
+        public CloudBlobContainer StorageContainer { get; set; }
```

Comment:
It is. I've confirmed with Sql team and they've agreed with this change. Current, this parameter doesn't support pipeline, so the change won't break any piping scenarios. CloudBlobContainer can be got with Get-AzureStorageContainer cmdlet too, I've updated examples in help.